.travis.yml

```diff
@@ -1,3 +1,5 @@
+install:
+- gem install cocoapods -v 1.0.0 && pod repo update && pod install
```

Comment:
While I think 1.0 makes pod install faster when there are no changes, this would add to build time unnecessarily when the Podfile hasn't changed. I'm working on an install script to improve this